src/Components/App/App.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
+import HomeContainer from '../HomeContainer/HomeContainer';
+import { ViewProvider } from '../../Context/ViewContext'
 import './App.css';
 import Header from '../Header/Header';
 
 function App() {
   return (
     <section className="App">
       <Header />
+      <ViewProvider>
+        <HomeContainer/>
+      </ViewProvider>
```

Comment:
I'm imagining that we will then wrap this provider in a `Route` for the `/` path, ya?